internal/observability/middleware.go

```diff
@@ -91,6 +91,10 @@ type Config struct {
 	// Scope to which metrics are emitted.
 	Scope *metrics.Scope
 
+	// List of metric tags being suppressed from being tagged on
```

Comment:
The comment should start with `MetricTagsBlocklist represents...` (and end with a period `.`)